src/System.Windows.Forms/tests/UnitTests/misc/CollectionHelperTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using Xunit;
+
+namespace System.Windows.Forms.misc.Tests
+{
+    public class CollectionHelperTests
+    {
+        [Fact]
+        public void HashtableCopyTo_throws_when_target_is_null()
+        {
+            var source = new Dictionary<string, string>();
+            var target = (object[])null;
+            var index = 0;
+
+            Assert.Throws<ArgumentNullException>(() => source.HashtableCopyTo(target, index));
```

Comment:
We could avoid local and follow below across these tests.

```suggestion
            Assert.Throws<ArgumentNullException>(() => source.HashtableCopyTo(target, index: 0));
```